### ISSUE TITLE
Backend issues

### ISSUE THREAD

**Issue by ishuonly** [ishuonly/ExpirationDateTracker#8]
Need to update the following backend issues:

[ ] Setting up APIs for integration of AI model with the backend.
[ ] Improper integration of MongoDB(inconsistent transferring of data from/to frontend)

**Comment by Aakash1312-cmd** [comment]
Can I fixture first issue?
If yes,Which framework will I use Django, Flask or FastAPI

**Comment by ishuonly** [comment]
Cool, I'll assign this issue to you. FastAPI has been used in this.

**Comment by Aakash1312-cmd** [comment]
Yeah sure I would love to work on this issue please provide me with the
information.

**Comment by ishuonly** [comment]
Please go through the readme. And let me know in case you need any specific information.

**Comment by Aakash1312-cmd** [comment]
Ok, I will do it.

**Comment by Aakash1312-cmd** [comment]
I was solving the issues raised but I am getting errors on VS Code as importing certain files but the server is running on the local host but its not displaying.

I want to contact you asap!!!..and I would like to connect with you on this matter
